hermeto/core/models/input.py

```diff
@@ -53,8 +53,18 @@ def show_error(error: "ErrorDict") -> str:
     return f"{header}\n{details}"
 
 
-# Supported package managers
-PackageManagerType = Literal["bundler", "cargo", "generic", "gomod", "npm", "pip", "rpm", "yarn"]
+PackageManagerType = Literal[
+    "bundler",
+    "cargo",
+    "generic",
+    "gomod",
+    "npm",
+    "pip",
+    "rpm",
+    "yarn",
+    # Add experimental package managers here with x- prefix, e.g. "x-foo"
```

Comment:
To make sure I understand the proposed change correctly -- is the only difference between a regular package manager and experimental is a prefix in its name (apparently with an assumption that while being experimental a PM would taint SBOMs)?

Reply:
Yes, spot on.